pallets/asset-manager/src/weights.rs

```diff
@@ -1,16 +1,16 @@
 // Copyright 2020-2022 Manta Network.
 // This file is part of Manta.
-
+//
 // Manta is free software: you can redistribute it and/or modify
 // it under the terms of the GNU General Public License as published by
 // the Free Software Foundation, either version 3 of the License, or
 // (at your option) any later version.
-
+//
```

Comment:
And add `//` to empty lines in https://github.com/Manta-Network/Manta/blob/manta/.github/resources/frame-weight-template.hbs

Reply:
adapted it, thanks!